aiconfig-docs/docs/gradio-workbook.md

```diff
@@ -0,0 +1,144 @@
+---
+sidebar_position: 21
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import constants from '@site/core/tabConstants';
+
+# Gradio Workbooks
+
+**[Gradio Workbooks](https://huggingface.co/spaces/lastmileai/gradio-workbook-template) provide an out-of-the-box playground for creating Hugging Face Spaces.**
+
+The playground provides instant model selection, chaining of models, and templates - no setup or code required!
+
+<div align="center">
+  <picture>
+    <img alt="workbook" src="https://github.com/lastmile-ai/aiconfig/assets/81494782/494523e1-23c6-4709-94cb-7ebe86873dfd" width="600"/>
+  </picture>
+</div>
+<br/>\
+Gradio Workbook unlocks new capabilities to HuggingFace Spaces users!
+
+1. **Build apps inspired by Spaces**. Gradio Workbook spaces are backed by an `aiconfig.json` file that stores prompts and model settings. Click **Download** to get the aiconfig file for your space. Use the `aiconfig.json` in your application code with the [AIConfig SDK](https://github.com/lastmile-ai/aiconfig).
+2. **Share read-only views of Spaces**. You can now share your workflow, prompts, and outputs, even as a space viewer. Check out this [space](https://huggingface.co/spaces/lastmileai/music_generator) and it's [read-only view](https://lastmileai.dev/aiconfig/cls2n2jpk00pnpe1n5xx8uub2)! Make edits to the space and click **Share Workbook** to get a read-only view of your own.
+
+## Quickstart
+
+### 1. Create a new space with Gradio Workbooks
+
+- **Start with a template**. Duplicate the [Gradio Workbook Quickstart space](https://huggingface.co/spaces/lastmileai/gradio-workbook-template)
+- **Create from scratch**. Create a [new Gradio SDK space](https://huggingface.co/new-space) and add these files to your space repo. To add files to your space repo, you can do so through the [Web UI](https://huggingface.co/docs/hub/repositories-getting-started#adding-files-to-a-repository-web-ui) or [terminal](https://huggingface.co/docs/hub/repositories-getting-started#terminal).
```

Comment:
Left-alignment is a gradio thing because of the prompt menu button missing padding. https://github.com/lastmile-ai/aiconfig/pull/1109 will fix

For the newlines, that actually wasn't possible before but should be possible (as well as using nice markdown) in the latest package now that https://github.com/lastmile-ai/aiconfig/pull/1094 landed and was published in latest gradio workbook package

Reply:
Making note to update the Quickstart space.